eldritch/items/unique.py

```diff
@@ -1,6 +1,6 @@
 import operator
 from eldritch import events, values
-from .base import Item, Weapon, OneshotWeapon
+from .base import Item, Weapon, OneshotWeapon, Tome
```

Comment:
Don't forget to add the new tomes to `__all__`, and also create them in the Create() functions. And also remove the TODO that references the tomes.